<?xml version="1.0" encoding="utf-8"?>
<TestRun id="19957ffc-f04c-402f-9944-45dd634a6a1f" name="kurt@DESKTOP 2024-11-14 17:13:34" runUser="DESKTOP\kurt" xmlns="http://microsoft.com/schemas/VisualStudio/TeamTest/2010">
  <Times creation="2024-11-14T17:13:34.6792745+10:00" queuing="2024-11-14T17:13:34.6792748+10:00" start="2024-11-14T17:13:33.5043223+10:00" finish="2024-11-14T17:13:34.7126551+10:00" />
  <TestSettings name="default" id="ae59725a-de07-4dec-8386-93855d314b71">
    <Deployment runDeploymentRoot="kurt_DESKTOP_2024-11-14_17_13_34" />
  </TestSettings>
  <Results>
    <UnitTestResult executionId="51984122-3540-4476-8301-b39a732199ed" testId="b8ede775-939f-1599-cdf5-655ae4036321" testName="SampleProject.xUnit.TestServiceTests.FailTest" computerName="DESKTOP" duration="00:00:00.0006567" startTime="2024-11-14T17:13:34.2891663+10:00" endTime="2024-11-14T17:13:34.2891665+10:00" testType="13cdc9d9-ddb5-4fa4-a97d-d965ccfc6d4b" outcome="Failed" testListId="8c84fa94-04c1-424b-9868-57a2d4851a1d" relativeResultsDirectory="51984122-3540-4476-8301-b39a732199ed">
      <Output>
        <StdOut>Running SampleProject.Tests.xUnit tests&#xD;
This test will fail</StdOut>
        <ErrorInfo>
          <Message>Assert.True() Failure&#xD;
Expected: True&#xD;
Actual:   False</Message>
          <StackTrace>   at SampleProject.xUnit.TestServiceTests.FailTest() in C:\Dev\LiquidTestReports\test\SampleProject\SampleProject.Tests.xUnit\TestServiceTests.cs:line 65&#xD;
   at System.RuntimeMethodHandle.InvokeMethod(Object target, Void** arguments, Signature sig, Boolean isConstructor)&#xD;
   at System.Reflection.MethodBaseInvoker.InvokeWithNoArgs(Object obj, BindingFlags invokeAttr)</StackTrace>
        </ErrorInfo>
      </Output>
    </UnitTestResult>
    <UnitTestResult executionId="1d9306dc-943a-4c7d-bed6-5de6d08a4ac5" testId="f1ad57ce-c1e1-68d8-f10c-211a14ef3768" testName="SampleProject.xUnit.TestServiceTests.PassingTest" computerName="DESKTOP" duration="00:00:00.0010885" startTime="2024-11-14T17:13:34.2886047+10:00" endTime="2024-11-14T17:13:34.2886048+10:00" testType="13cdc9d9-ddb5-4fa4-a97d-d965ccfc6d4b" outcome="Passed" testListId="8c84fa94-04c1-424b-9868-57a2d4851a1d" relativeResultsDirectory="1d9306dc-943a-4c7d-bed6-5de6d08a4ac5">
      <Output>
        <StdOut>Running SampleProject.Tests.xUnit tests</StdOut>
      </Output>
    </UnitTestResult>
    <UnitTestResult executionId="de190ceb-3cf3-4b58-978c-bd7cf3a8e587" testId="dc4d4b72-ac66-1bab-934d-d3322145984c" testName="SampleProject.xUnit.TestServiceTests.TestTheory(expected: True)" computerName="DESKTOP" duration="00:00:00.0059253" startTime="2024-11-14T17:13:34.2797935+10:00" endTime="2024-11-14T17:13:34.2797936+10:00" testType="13cdc9d9-ddb5-4fa4-a97d-d965ccfc6d4b" outcome="Passed" testListId="8c84fa94-04c1-424b-9868-57a2d4851a1d" relativeResultsDirectory="de190ceb-3cf3-4b58-978c-bd7cf3a8e587">
      <Output>
        <StdOut>Running SampleProject.Tests.xUnit tests</StdOut>
      </Output>
    </UnitTestResult>
    <UnitTestResult executionId="8ccac91c-b7ff-4a7f-b85a-a1f0421daa61" testId="2eb95158-b57c-2a18-89f9-5ccdf45dcabc" testName="SampleProject.xUnit.TestServiceTests.TestThrowingException" computerName="DESKTOP" duration="00:00:00.0072247" startTime="2024-11-14T17:13:34.2792616+10:00" endTime="2024-11-14T17:13:34.2792620+10:00" testType="13cdc9d9-ddb5-4fa4-a97d-d965ccfc6d4b" outcome="Failed" testListId="8c84fa94-04c1-424b-9868-57a2d4851a1d" relativeResultsDirectory="8ccac91c-b7ff-4a7f-b85a-a1f0421daa61">
      <Output>
        <StdOut>Running SampleProject.Tests.xUnit tests</StdOut>
        <ErrorInfo>
          <Message>System.Exception : Pretty good exception</Message>
          <StackTrace>   at SampleProject.TestService.GetException() in C:\Dev\LiquidTestReports\test\SampleProject\SampleProject\TestService.cs:line 19&#xD;
   at SampleProject.xUnit.TestServiceTests.TestThrowingException() in C:\Dev\LiquidTestReports\test\SampleProject\SampleProject.Tests.xUnit\TestServiceTests.cs:line 54&#xD;
   at System.RuntimeMethodHandle.InvokeMethod(Object target, Void** arguments, Signature sig, Boolean isConstructor)&#xD;
   at System.Reflection.MethodBaseInvoker.InvokeWithNoArgs(Object obj, BindingFlags invokeAttr)</StackTrace>
        </ErrorInfo>
      </Output>
    </UnitTestResult>
    <UnitTestResult executionId="83f2eda3-cfe1-4b24-8116-3af9b2295f7d" testId="63207abc-d9b4-7a30-6461-919c9a06a2b6" testName="SampleProject.xUnit.TestServiceTests.SkipTest" computerName="DESKTOP" duration="00:00:00.0010000" startTime="2024-11-14T17:13:34.2443268+10:00" endTime="2024-11-14T17:13:34.2443553+10:00" testType="13cdc9d9-ddb5-4fa4-a97d-d965ccfc6d4b" outcome="NotExecuted" testListId="8c84fa94-04c1-424b-9868-57a2d4851a1d" relativeResultsDirectory="83f2eda3-cfe1-4b24-8116-3af9b2295f7d">
      <Output>
        <ErrorInfo>
          <Message>Skipped</Message>
        </ErrorInfo>
      </Output>
    </UnitTestResult>
    <UnitTestResult executionId="bef7bf32-0330-4b4b-b143-ab108ca99d5a" testId="1a5e29b3-7ef1-5c3d-f997-77c67b0b2a38" testName="SampleProject.NUnit.Attachment.AttachImage" computerName="DESKTOP" duration="00:00:00.0063242" startTime="2024-11-14T17:13:34.2612548+10:00" endTime="2024-11-14T17:13:34.2612552+10:00" testType="13cdc9d9-ddb5-4fa4-a97d-d965ccfc6d4b" outcome="Passed" testListId="8c84fa94-04c1-424b-9868-57a2d4851a1d" relativeResultsDirectory="bef7bf32-0330-4b4b-b143-ab108ca99d5a">
      <Output>
        <StdOut>Not actually an attachment, just a placeholder to make the results the same.</StdOut>
      </Output>
    </UnitTestResult>
    <UnitTestResult executionId="03ed3aae-93df-43d2-a437-8e635727ce6f" testId="4eee5b76-ac5c-892d-7905-27554828341a" testName="SampleProject.xUnit.TestServiceTests.TestTheory(expected: False)" computerName="DESKTOP" duration="00:00:00.0037869" startTime="2024-11-14T17:13:34.2884486+10:00" endTime="2024-11-14T17:13:34.2884488+10:00" testType="13cdc9d9-ddb5-4fa4-a97d-d965ccfc6d4b" outcome="Failed" testListId="8c84fa94-04c1-424b-9868-57a2d4851a1d" relativeResultsDirectory="03ed3aae-93df-43d2-a437-8e635727ce6f">
      <Output>
        <StdOut>Running SampleProject.Tests.xUnit tests</StdOut>
        <ErrorInfo>
          <Message>Assert.Equal() Failure: Values differ&#xD;
Expected: False&#xD;
Actual:   True</Message>
          <StackTrace>   at SampleProject.xUnit.TestServiceTests.TestTheory(Boolean expected) in C:\Dev\LiquidTestReports\test\SampleProject\SampleProject.Tests.xUnit\TestServiceTests.cs:line 29&#xD;
   at InvokeStub_TestServiceTests.TestTheory(Object, Span`1)&#xD;
   at System.Reflection.MethodBaseInvoker.InvokeWithOneArg(Object obj, BindingFlags invokeAttr, Binder binder, Object[] parameters, CultureInfo culture)</StackTrace>
        </ErrorInfo>
      </Output>
    </UnitTestResult>
  </Results>
  <TestDefinitions>
    <UnitTest name="SampleProject.xUnit.TestServiceTests.SkipTest" storage="c:\dev\liquidtestreports\test\sampleproject\sampleproject.tests.xunit\bin\debug\net9.0\sampleproject.xunit.dll" id="63207abc-d9b4-7a30-6461-919c9a06a2b6">
      <Execution id="83f2eda3-cfe1-4b24-8116-3af9b2295f7d" />
      <TestMethod codeBase="C:\Dev\LiquidTestReports\test\SampleProject\SampleProject.Tests.xUnit\bin\Debug\net9.0\SampleProject.xUnit.dll" adapterTypeName="executor://xunit/VsTestRunner2/netcoreapp" className="SampleProject.xUnit.TestServiceTests" name="SkipTest" />
    </UnitTest>
    <UnitTest name="SampleProject.xUnit.TestServiceTests.TestTheory(expected: True)" storage="c:\dev\liquidtestreports\test\sampleproject\sampleproject.tests.xunit\bin\debug\net9.0\sampleproject.xunit.dll" id="dc4d4b72-ac66-1bab-934d-d3322145984c">
      <Execution id="de190ceb-3cf3-4b58-978c-bd7cf3a8e587" />
      <TestMethod codeBase="C:\Dev\LiquidTestReports\test\SampleProject\SampleProject.Tests.xUnit\bin\Debug\net9.0\SampleProject.xUnit.dll" adapterTypeName="executor://xunit/VsTestRunner2/netcoreapp" className="SampleProject.xUnit.TestServiceTests" name="TestTheory" />
    </UnitTest>
    <UnitTest name="SampleProject.xUnit.TestServiceTests.TestThrowingException" storage="c:\dev\liquidtestreports\test\sampleproject\sampleproject.tests.xunit\bin\debug\net9.0\sampleproject.xunit.dll" id="2eb95158-b57c-2a18-89f9-5ccdf45dcabc">
      <Execution id="8ccac91c-b7ff-4a7f-b85a-a1f0421daa61" />
      <TestMethod codeBase="C:\Dev\LiquidTestReports\test\SampleProject\SampleProject.Tests.xUnit\bin\Debug\net9.0\SampleProject.xUnit.dll" adapterTypeName="executor://xunit/VsTestRunner2/netcoreapp" className="SampleProject.xUnit.TestServiceTests" name="TestThrowingException" />
    </UnitTest>
    <UnitTest name="SampleProject.xUnit.TestServiceTests.FailTest" storage="c:\dev\liquidtestreports\test\sampleproject\sampleproject.tests.xunit\bin\debug\net9.0\sampleproject.xunit.dll" id="b8ede775-939f-1599-cdf5-655ae4036321">
      <Execution id="51984122-3540-4476-8301-b39a732199ed" />
      <TestMethod codeBase="C:\Dev\LiquidTestReports\test\SampleProject\SampleProject.Tests.xUnit\bin\Debug\net9.0\SampleProject.xUnit.dll" adapterTypeName="executor://xunit/VsTestRunner2/netcoreapp" className="SampleProject.xUnit.TestServiceTests" name="FailTest" />
    </UnitTest>
    <UnitTest name="SampleProject.xUnit.TestServiceTests.TestTheory(expected: False)" storage="c:\dev\liquidtestreports\test\sampleproject\sampleproject.tests.xunit\bin\debug\net9.0\sampleproject.xunit.dll" id="4eee5b76-ac5c-892d-7905-27554828341a">
      <Execution id="03ed3aae-93df-43d2-a437-8e635727ce6f" />
      <TestMethod codeBase="C:\Dev\LiquidTestReports\test\SampleProject\SampleProject.Tests.xUnit\bin\Debug\net9.0\SampleProject.xUnit.dll" adapterTypeName="executor://xunit/VsTestRunner2/netcoreapp" className="SampleProject.xUnit.TestServiceTests" name="TestTheory" />
    </UnitTest>
    <UnitTest name="SampleProject.NUnit.Attachment.AttachImage" storage="c:\dev\liquidtestreports\test\sampleproject\sampleproject.tests.xunit\bin\debug\net9.0\sampleproject.xunit.dll" id="1a5e29b3-7ef1-5c3d-f997-77c67b0b2a38">
      <Execution id="bef7bf32-0330-4b4b-b143-ab108ca99d5a" />
      <TestMethod codeBase="C:\Dev\LiquidTestReports\test\SampleProject\SampleProject.Tests.xUnit\bin\Debug\net9.0\SampleProject.xUnit.dll" adapterTypeName="executor://xunit/VsTestRunner2/netcoreapp" className="SampleProject.NUnit.Attachment" name="AttachImage" />
    </UnitTest>
    <UnitTest name="SampleProject.xUnit.TestServiceTests.PassingTest" storage="c:\dev\liquidtestreports\test\sampleproject\sampleproject.tests.xunit\bin\debug\net9.0\sampleproject.xunit.dll" id="f1ad57ce-c1e1-68d8-f10c-211a14ef3768">
      <Execution id="1d9306dc-943a-4c7d-bed6-5de6d08a4ac5" />
      <TestMethod codeBase="C:\Dev\LiquidTestReports\test\SampleProject\SampleProject.Tests.xUnit\bin\Debug\net9.0\SampleProject.xUnit.dll" adapterTypeName="executor://xunit/VsTestRunner2/netcoreapp" className="SampleProject.xUnit.TestServiceTests" name="PassingTest" />
    </UnitTest>
  </TestDefinitions>
  <TestEntries>
    <TestEntry testId="b8ede775-939f-1599-cdf5-655ae4036321" executionId="51984122-3540-4476-8301-b39a732199ed" testListId="8c84fa94-04c1-424b-9868-57a2d4851a1d" />
    <TestEntry testId="f1ad57ce-c1e1-68d8-f10c-211a14ef3768" executionId="1d9306dc-943a-4c7d-bed6-5de6d08a4ac5" testListId="8c84fa94-04c1-424b-9868-57a2d4851a1d" />
    <TestEntry testId="dc4d4b72-ac66-1bab-934d-d3322145984c" executionId="de190ceb-3cf3-4b58-978c-bd7cf3a8e587" testListId="8c84fa94-04c1-424b-9868-57a2d4851a1d" />
    <TestEntry testId="2eb95158-b57c-2a18-89f9-5ccdf45dcabc" executionId="8ccac91c-b7ff-4a7f-b85a-a1f0421daa61" testListId="8c84fa94-04c1-424b-9868-57a2d4851a1d" />
    <TestEntry testId="63207abc-d9b4-7a30-6461-919c9a06a2b6" executionId="83f2eda3-cfe1-4b24-8116-3af9b2295f7d" testListId="8c84fa94-04c1-424b-9868-57a2d4851a1d" />
    <TestEntry testId="1a5e29b3-7ef1-5c3d-f997-77c67b0b2a38" executionId="bef7bf32-0330-4b4b-b143-ab108ca99d5a" testListId="8c84fa94-04c1-424b-9868-57a2d4851a1d" />
    <TestEntry testId="4eee5b76-ac5c-892d-7905-27554828341a" executionId="03ed3aae-93df-43d2-a437-8e635727ce6f" testListId="8c84fa94-04c1-424b-9868-57a2d4851a1d" />
  </TestEntries>
  <TestLists>
    <TestList name="Results Not in a List" id="8c84fa94-04c1-424b-9868-57a2d4851a1d" />
    <TestList name="All Loaded Results" id="19431567-8539-422a-85d7-44ee4e166bda" />
  </TestLists>
  <ResultSummary outcome="Failed">
    <Counters total="7" executed="6" passed="3" failed="3" error="0" timeout="0" aborted="0" inconclusive="0" passedButRunAborted="0" notRunnable="0" notExecuted="0" disconnected="0" warning="0" completed="0" inProgress="0" pending="0" />
    <Output>
      <StdOut>[xUnit.net 00:00:00.00] xUnit.net VSTest Adapter v2.8.2+699d445a1a (64-bit .NET 9.0.0)&#xD;
[xUnit.net 00:00:00.11]   Discovering: SampleProject.xUnit&#xD;
[xUnit.net 00:00:00.19]   Discovered:  SampleProject.xUnit&#xD;
[xUnit.net 00:00:00.20]   Starting:    SampleProject.xUnit&#xD;
[xUnit.net 00:00:00.25]       Skipped&#xD;
[xUnit.net 00:00:00.28]       System.Exception : Pretty good exception&#xD;
[xUnit.net 00:00:00.28]       Stack Trace:&#xD;
[xUnit.net 00:00:00.28]         C:\Dev\LiquidTestReports\test\SampleProject\SampleProject\TestService.cs(19,0): at SampleProject.TestService.GetException()&#xD;
[xUnit.net 00:00:00.28]         C:\Dev\LiquidTestReports\test\SampleProject\SampleProject.Tests.xUnit\TestServiceTests.cs(54,0): at SampleProject.xUnit.TestServiceTests.TestThrowingException()&#xD;
[xUnit.net 00:00:00.28]            at System.RuntimeMethodHandle.InvokeMethod(Object target, Void** arguments, Signature sig, Boolean isConstructor)&#xD;
[xUnit.net 00:00:00.28]            at System.Reflection.MethodBaseInvoker.InvokeWithNoArgs(Object obj, BindingFlags invokeAttr)&#xD;
[xUnit.net 00:00:00.28]       Output:&#xD;
[xUnit.net 00:00:00.28]         Running SampleProject.Tests.xUnit tests&#xD;
[xUnit.net 00:00:00.29]       Assert.Equal() Failure: Values differ&#xD;
[xUnit.net 00:00:00.29]       Expected: False&#xD;
[xUnit.net 00:00:00.29]       Actual:   True&#xD;
[xUnit.net 00:00:00.29]       Stack Trace:&#xD;
[xUnit.net 00:00:00.29]         C:\Dev\LiquidTestReports\test\SampleProject\SampleProject.Tests.xUnit\TestServiceTests.cs(29,0): at SampleProject.xUnit.TestServiceTests.TestTheory(Boolean expected)&#xD;
[xUnit.net 00:00:00.29]            at InvokeStub_TestServiceTests.TestTheory(Object, Span`1)&#xD;
[xUnit.net 00:00:00.29]            at System.Reflection.MethodBaseInvoker.InvokeWithOneArg(Object obj, BindingFlags invokeAttr, Binder binder, Object[] parameters, CultureInfo culture)&#xD;
[xUnit.net 00:00:00.29]       Output:&#xD;
[xUnit.net 00:00:00.29]         Running SampleProject.Tests.xUnit tests&#xD;
[xUnit.net 00:00:00.29]       Assert.True() Failure&#xD;
[xUnit.net 00:00:00.29]       Expected: True&#xD;
[xUnit.net 00:00:00.29]       Actual:   False&#xD;
[xUnit.net 00:00:00.29]       Stack Trace:&#xD;
[xUnit.net 00:00:00.29]         C:\Dev\LiquidTestReports\test\SampleProject\SampleProject.Tests.xUnit\TestServiceTests.cs(65,0): at SampleProject.xUnit.TestServiceTests.FailTest()&#xD;
[xUnit.net 00:00:00.29]            at System.RuntimeMethodHandle.InvokeMethod(Object target, Void** arguments, Signature sig, Boolean isConstructor)&#xD;
[xUnit.net 00:00:00.29]            at System.Reflection.MethodBaseInvoker.InvokeWithNoArgs(Object obj, BindingFlags invokeAttr)&#xD;
[xUnit.net 00:00:00.29]       Output:&#xD;
[xUnit.net 00:00:00.29]         Running SampleProject.Tests.xUnit tests&#xD;
[xUnit.net 00:00:00.29]         This test will fail&#xD;
[xUnit.net 00:00:00.30]   Finished:    SampleProject.xUnit&#xD;
Test 'SampleProject.xUnit.TestServiceTests.SkipTest' was skipped in the test run.&#xD;
</StdOut>
    </Output>
    <RunInfos>
      <RunInfo computerName="DESKTOP" outcome="Warning" timestamp="2024-11-14T17:13:34.2448372+10:00">
        <Text>[xUnit.net 00:00:00.25]     SampleProject.xUnit.TestServiceTests.SkipTest [SKIP]</Text>
      </RunInfo>
      <RunInfo computerName="DESKTOP" outcome="Error" timestamp="2024-11-14T17:13:34.2724377+10:00">
        <Text>[xUnit.net 00:00:00.28]     SampleProject.xUnit.TestServiceTests.TestThrowingException [FAIL]</Text>
      </RunInfo>
      <RunInfo computerName="DESKTOP" outcome="Error" timestamp="2024-11-14T17:13:34.2886183+10:00">
        <Text>[xUnit.net 00:00:00.29]     SampleProject.xUnit.TestServiceTests.TestTheory(expected: False) [FAIL]</Text>
      </RunInfo>
      <RunInfo computerName="DESKTOP" outcome="Error" timestamp="2024-11-14T17:13:34.2891694+10:00">
        <Text>[xUnit.net 00:00:00.29]     SampleProject.xUnit.TestServiceTests.FailTest [FAIL]</Text>
      </RunInfo>
    </RunInfos>
  </ResultSummary>
</TestRun>